### ISSUE TITLE
utf-8 error

### ISSUE THREAD

**Issue by ktkchn** [devsnd/tinytag#34]
I get this error with some unicode song titles.MP4Box was used to pack the m4a file
and the title is "Cold Water (feat. Justin Bieber & MØ)".
UnicodeDecodeError: 'utf8' codec can't decode byte 0x92 in position 34: invalid start byte
Any help is appreciated.

**Comment by devsnd** [comment]
Hello @ktkchn,

thanks for reporting! Can you please upload the mp4 file somewhere so i can reproduce the bug? thanks!

**Comment by ktkchn** [comment]
hello @devsnd 

I have uploaded the m4a file here
https://drive.google.com/file/d/0B4rXGVsThTq6UjBEcjZCX29aWFU/view?usp=sharing

thanks

**Comment by devsnd** [comment]
Hey @ktkchn,

I have reproduced the bug and also found out why TinyTag crashes; The file you provided is malformed and does not adhere to the spec. [Apples specification for MP4](https://developer.apple.com/library/mac/documentation/QuickTime/QTFF/Metadata/Metadata.html#//apple_ref/doc/uid/TP40000939-CH1-SW34) allows strings only to be encoded using UTF-8. The file you provided seems to use ISO-8859-1 (also called latin1). Since there is no way to find out which encoding a byte-sequence was encoded in, I cannot just decode your file in a different manner.

I have added a patch that will replace all characters that cannot be decoded with `�`, so that TinyTag will not crash anymore.
